extensions/git/package.json

```diff
@@ -2759,13 +2759,8 @@
           "default": false
         },
         "git.inputValidation": {
-          "type": "string",
-          "enum": [
-            "always",
-            "warn",
-            "off"
-          ],
-          "default": "off",
+          "type": "boolean",
```

Comment:
You could just keep it as an enum with two options? Is much easier if you ever want to go back and add more